docs/content/en/blog/releases/v0.40.0.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Release v0.40.0"
+linkTitle: "Release v0.40.0"
+date: 2022-11-30
+description: >
+ Release v0.40.0
+---
+
+## Changes since v0.39.0
+### Notable Changes
+
+* Enable to restart Piped from the PipeCD console [#3988](https://github.com/pipe-cd/pipecd/pull/3988)
+* Allow config MySQL database via config URL ([#4017](https://github.com/pipe-cd/pipecd/pull/4017))
+* Fix K8s resources are not pruned even if running Quick Sync ([#3991](https://github.com/pipe-cd/pipecd/pull/3991))
+
+### Internal Changes
+
+* Release v0.40.0
```

Comment:
```suggestion
```

Reply:
oh, it's too late.

Reply:
We can remove it later with docs stub fix 👍